### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by Taras-Novitskyi**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
+
+footer {
+	background: red;
+}
+
 header {
```
Comment (oleksandr-vasylenko) [inline css/styles.css]: Please move the header to the top

File: index.html [hunk 2]
```diff
@@ -1,11 +1,10 @@
-<!DOCTYPE html>
-<html lang="en">
+
```
Comment (oleksandr-vasylenko) [inline index.html]: Please add document and lang tags

**Review by oleksandr-vasylenko — CHANGES_REQUESTED** [review]
Please fix

File: css/styles.css (hunk 1 above)
Comment (Taras-Novitskyi) [reply css/styles.css]: fixed

File: index.html (hunk 2 above)
Comment (Taras-Novitskyi) [reply index.html]: fixed

**Review by oleksandr-vasylenko — APPROVED** [review]

**Pull request merged**